### ISSUE TITLE
[EXE] Introduction to Bayesian Statistical Modeling

### ISSUE THREAD

**Issue by des137** [gimseng/99-ML-Learning-Projects#143]
#### Learning Goals
- Learn the concepts behind Bayesian statistical modeling with PyMC3
- Compare and contrast the frequentist and the Bayesian inference approaches

### Exercise Statement
Compare the stochastic gradient based frequentist's method vs. probabilistic Bayesian method to solve a simple linear regression problem and interpret and contrast the outcomes from both the approaches.

### Prerequisites 
- Basic knowledge about linear regression (#43)
- Basic knowledge of Bayesian approach to probability

### Data source/summary:
This comparison can be performed on a simulated data generated through _sklearn.make_regression_ function.

### (Optional) Further Links/Credits to Relevant Resources:
https://docs.pymc.io/
